src/main/java/seedu/address/logic/commands/alias/UnaliasCommand.java

```diff
@@ -0,0 +1,55 @@
+package seedu.address.logic.commands.alias;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.logic.commands.Command;
+import seedu.address.logic.commands.CommandResult;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+
+/**
+ * Deletes the alias with the specified name.
+ */
+public class UnaliasCommand extends Command {
+
+    public static final String COMMAND_WORD = "unalias";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Delete alias with the specified name.\n"
+            + "Parameters: a/ALIAS\n"
+            + "Example: unalias a/findBob";
+
+    public static final String MESSAGE_SUCCESS = "Alias deleted: %1$s";
+
+    public static final String MESSAGE_FAILURE = "Alias not found";
+
+    private String aliasName;
+
+    /**
+     * Creates an UnaliasCommand with the specified {@code aliasName}
+     */
+    public UnaliasCommand(String aliasName) {
+        requireNonNull(aliasName);
+        this.aliasName = aliasName.trim();
+    }
+
+    @Override
```

Comment:
It might be good to document this method too, even though it's an overridden method because each command is different.

Reply:
Done!